lib/dialects/postgres/query-generator.js

```diff
@@ -320,7 +320,7 @@ module.exports = (function() {
       when  = when || 'unique_violation';
 
       var body = '<%= main %> EXCEPTION WHEN <%= when %> THEN <%= then %>;';
-      body = Utils._.template(body, {
+      body = Utils._.template(body)({
```

Comment:
The biggest change was the change from `template(string, values)` to `template(string)(values)`